src/20240118_StkGHO_Activation/StkGHO_Activation_20240118.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IAggregatedStakeToken} from 'stake-token/src/contracts/IAggregatedStakeToken.sol';
+import {DistributionTypes} from 'stake-token/src/contracts/lib/DistributionTypes.sol';
+import {IAaveDistributionManager} from 'stake-token/src/contracts/IAaveDistributionManager.sol';
+import {AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+
+/**
+ * @title StkGHO Activation
+ * @author the3d.eth
+ * - Snapshot: 'https://snapshot.org/#/aave.eth/proposal/0x4bc99a842adab6cdd8c7d5c7a787ee4c0056be554fde0d008d53b45b3e795065
+ * - Discussion: https://governance.aave.com/t/arfc-upgrade-safety-module-with-stkgho/15635
+ */
+contract StkGHO_Activation_20240118 is IProposalGenericExecutor {
+  uint256 public constant STKGHO_EMISSION_PER_SECOND = 5787037037037037; // 50 AAVE/day
+  uint256 public constant DISTRIBUTION_DURATION = 3*30*86400; // three months
+  address public constant STKGHO_PROXY = 0x1a88Df1cFe15Af22B3c4c783D4e6F7F9e0C1885d;
+
+  function execute() external {
+     IAggregatedStakeToken(STKGHO_PROXY).setDistributionEnd(
+      block.timestamp + DISTRIBUTION_DURATION
+    );
+    DistributionTypes.AssetConfigInput[]
+      memory enableConfigs = new DistributionTypes.AssetConfigInput[](1);
+    enableConfigs[0] = DistributionTypes.AssetConfigInput({
+      emissionPerSecond: uint128(STKGHO_EMISSION_PER_SECOND),
+      totalStaked: 0, // it's overwritten internally
+      underlyingAsset: AaveV3Ethereum.GHO_TOKEN
```

Comment:
i'm relatively sure this is wrong - underlyingAsset i think needs to be the STK_GHO, no?